src/Caching/Storages/NewMemcachedStorage.php

```diff
@@ -111,6 +111,38 @@ public function read($key)
 
 
 	/**
+	 * Read from cache.
+	 * @param  string key
+	 * @return array key => value pairs, missing items are omitted
+	 */
+	public function bulkRead(array $keys)
+	{
+		$keys = array_combine(array_map(function ($key) {
+			return urlencode($this->prefix . $key);
+		}, $keys), $keys);
+		$metas = $this->memcached->getMulti(array_keys($keys));
```

Comment:
Replace with

~~~php
$prefixedKeys = array_map(function ($key) { return urlencode($this->prefix . $key); });
$keys = array_combine($prefixedKeys, $keys); // TODO: can we remove this?
$metas = $this->memcached->getMulti($prefixedKeys);
~~~